packages/compiler/src/experimental/index.ts

```diff
@@ -7,6 +7,7 @@ export {
   MutatorRecord as unsafe_MutatorRecord,
   MutatorReplaceFn as unsafe_MutatorReplaceFn,
   MutatorWithNamespace as unsafe_MutatorWithNamespace,
+  isMutableType as unsafe_isMutableType,
```

Comment:
I don't see where this is used

Reply:
So we were using this in an earlier draft but realized we needed a more specific check than `isMutableType`. Opposed to just leaving this export in? Seems like it's an oversight not to include this in the unsafe surface because it's the only type guard that exists for MutableType.

Reply:
I guess that's fine